Notoday/NotePage.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  NotePage.swift
+//  Notoday
+//
+//  Created by Jamie Joung on 11/7/22.
+//
+
+
+import Foundation
+import SwiftUI
+import CoreData
+
+
+//enum Emotion: String, Identifiable, CaseIterable {
+//
+//    var id: UUID {
+//        return UUID()
+//    }
+//
+//    case happy = "Happy"
+//    case satisfied = "Satisfied"
+//    case neutral = "Neutral"
+//    case tired = "Tired"
+//    case upset = "Upset"
+//}
+//
+//extension Emotion {
+//    var title: String {
```

Comment:
Ok for this one, you would just return your `rawValue` from the Emotion because that's what you defined up above in lines 20-24.